__tests__/full-setup.test.js

```diff
@@ -134,6 +135,42 @@ describe.each`
         });
     });
 
+    test('select specific fields only', () => {
+      const db = firebase.firestore();
+      // Example from documentation:
+      // https://firebase.google.com/docs/firestore/quickstart#read_data
```

Comment:
This doesn't look like it points to example with `select`. Maybe the docs have been updated?